dkist/net/attrs_values.py

```diff
@@ -105,6 +106,11 @@ def attempt_local_update(*, timeout: int = 1, user_file: Path = None, silence_er
     try:
         _fetch_values_to_file(user_file, timeout=timeout)
         success = True
+    except URLError as urlerr:
```

Comment:
This is a good addition, but I don't think it solves the general problem as we could still end up in a situation where we later try to access a file we just deleted. We need to rework the logic a bit more I think, to either not delete the file at all (maybe download to a tmp location and then move on success) or to revert to the built in one on fail but that leads to using an older file.

Reply:
I'm not sure this is true, because the unlink command is deliberately in the second except block so the file won't ever have been deleted if the first one catches.